controllers/users.js

```diff
@@ -0,0 +1,97 @@
+const md5 = require("md5");
+const db = require("../models");
+
+module.exports = {
+  getUsers: async (req, res) => {
+    try {
+      let users = await db.User.findAll();
+      res.json(users);
+    } catch (err) {
+      res.send(err);
+    }
+  },
+  createUser: async (req, res) => {
+    try {
+      let newUser = await db.User.create({
+        username: req.body.username,
+        password: md5(req.body.password),
+        confirm_password: md5(req.body.confirm_password),
+        email: req.body.email,
+        firstname: req.body.firstname,
+        lastname: req.body.lastname,
+      });
+      res.json(newUser);
+    } catch (err) {
+      res.send(err);
+    }
+  },
+  userLogin: async (req, res) => {
+    try {
+      let loginDetails = await db.User.findOne({
+        where: {
+          username: req.body.username,
+          password: md5(req.body.password),
+        },
+      });
+      let tokenDetails = await db.Accesstoken.findOne({
+        where: {
+          user_id: loginDetails.id,
+        },
+      });
+      if (tokenDetails === null) {
```

Comment:
no need to compare with ===. us direct true or false.null is considered as false